lib/components/artist/biography.js

```diff
@@ -21,7 +21,7 @@ class Biography extends React.Component {
 
   render() {
     const artist = this.props.artist;
-    if (artist.blurb.length === 0 && artist.bio.length === 0) { return null; }
+    if (!artist.blurb && !artist.bio) { return null; }
```

Comment:
This was causing an error in RN when I tested it with the artist `ry-david-bradley`, because the blurb was `null`. Would there be a case where there was a blurb, but it'd be empty? If so we would need both checks I'm assuming..